File: PSet1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void print_chars(char ch, int num);

int main(void)
{
    // Get an input for height
    // Requires height > 0
    int height = 0;
    while (!(height >= 1 && height <= 8))
    {
        height = get_int("Height: ");
    }

    // Print the pyramids
    for (int line = 1; line <= height; line++)
    {
        // Print a line
        print_chars(' ', height - line);
        print_chars('#', line);
        print_chars(' ', 2);
        print_chars('#', line);
        //print_chars(' ', height - line);
        printf("\n");
    }
}

// Print the character ch for num times
void print_chars(char ch, int num)
{
    for (int i = 0; i < num; i++)
    {
        printf("%c", ch);
    }
}

File: PSet6/sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")

    # Check validity
    if height >= 1 and height <= 8:
        break

# Print the pyramid
for i in range(1, height + 1):
    print(" " * (height - i) + "#" * i + "  " + "#" * i)